cdap-standalone/bin/cdap.bat

```diff
@@ -332,6 +332,8 @@ attrib +h %~dsp0MyProg.pid >NUL
 
 :SearchLogs
 <nul (SET /p _tmp=.)
+REM Sleep for 1 seconds to prevent spinning on fast machines
+PING 127.0.0.1 -n 2 > NUL 2>&1

```

Comment:
There's one in the "2003 Resource Kit" (see https://ss64.com/nt/sleep.html) but ``ping`` is regarded as superior because it won't interfere with the other processes, like our Java process starting in the background.